packages/legacy/core/App/modules/openid/metadata.tsx

```diff
@@ -0,0 +1,53 @@
+import type { W3cCredentialRecord, SdJwtVcRecord } from '@credo-ts/core'
+import type { OpenId4VciCredentialSupported, OpenId4VciIssuerMetadataDisplay } from '@credo-ts/openid4vc'
+import type { EndpointMetadataResult } from '@sphereon/oid4vci-common'
+
+export interface OpenId4VcCredentialMetadata {
+  credential: {
+    display?: OpenId4VciCredentialSupported['display']
+    order?: OpenId4VciCredentialSupported['order']
+  }
+  issuer: {
+    display?: OpenId4VciIssuerMetadataDisplay[]
+    id: string
+  }
+}
+
+const openId4VcCredentialMetadataKey = '_paradym/openId4VcCredentialMetadata'
```

Comment:
Does this introduce a dependency on Paradym?

Reply:
No, but you are right, I should introduce a different key

Reply:
done